src/components/board/board.ts

```diff
@@ -1,10 +1,18 @@
+import { type Duration } from 'google-protobuf/google/protobuf/duration_pb';
+import pb from '../../gen/component/board/v1/board_pb';
 import type { Resource, StructType } from '../../types';
 
 interface Status {
   analogs: Record<string, number>;
   digitalInterrupts: Record<string, number>;
 }
 
+type ValueOf<T> = T[keyof T];
+export const { PowerMode } = pb;
+export type PowerMode = ValueOf<typeof pb.PowerMode>;
+
+export type DurationOptions = Duration.AsObject;
```

Comment:
[minor] see comment related to naming

```suggestion
export type Duration = PBDuration.AsObject;
```